client/package.json

```diff
@@ -18,6 +18,8 @@
     "@sentry/browser": "^6.19.7",
     "@sentry/react": "^6.19.7",
     "@types/amplitude-js": "^8.16.1",
+    "@types/dompurify": "^3.0.5",
+    "dompurify": "^3.1.6",
```

Comment:
For å rense HTML for andre element enn <strong> og <em> (som er dei vi trenger for å kunne formatere tekst. Feks. bla bla bla oppfyller _ikke_ kravene for bla bla)